projects/pyre/build.mk

```diff
@@ -0,0 +1,9 @@
+PYRE_CHECK_DONE := build/host/pyre-check.done
```

Comment:
Could you add a copyright header like this one ?
https://github.com/facebookexperimental/ExtendedAndroidTools/blob/main/projects/stdc%2B%2Bfs/build.mk#L1

Reply:
Let me add that right away